diesel_cli/src/main.rs

```diff
@@ -1,5 +1,20 @@
+// Built-in Lints
+#![deny(warnings, missing_copy_implementations)]
+
+// Clippy lints
+#![cfg_attr(feature = "clippy", allow(unstable_features))]
```

Comment:
Not sure. Let's just try it and see what happens. I'd put it in the project root directory and call it `make_crate_pedantic.rs`